### PR TITLE
XCode 9.1 and iOS 11 support + Unit test refactoring

### PR DESCRIPTION
**Pull request by federicocappelli-je**
https://jira.just-eat.net/browse/CPD-8379

### CODE REVIEW

**Comment by edrutter-je** [comment]
Have these changes been verified to work with Xcode 8.x.x and pre iOS 11?

File: Example/JustTrack/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    @IBAction func fireEventsHandler(sender: UIButton?) {
+        
+        let trackingService: JETracking = configureJustTrack()
+        
+        trackingService.trackEvent(JEEventUser(action: "UseLogiIn", response: "success", extra: "Additional info"))
```
Comment (edrutter-je) [inline Example/JustTrack/ViewController.swift]: Is "UseLogiIn" a typo?

**Comment by federicocappelli-je** [comment]
tested in XCode 8.x.x

**Review by edrutter-je — APPROVED** [review]
👍  LGTM

File: Example/Tests/TrackerTests.swift [hunk 2]
```diff
@@ -141,22 +178,23 @@ class TrackerTests: XCTestCase {
...
-        trackerService.loadCustomTracker(tracker2)
+        trackerService.loadCustomTracker(tracker1!)
+        trackerService.loadCustomTracker(tracker2!)
         trackerService.deliveryType = .immediate
+        trackerService.enable()
         
         // WHEN we ask JustTrack to track the event
         trackerService.trackEvent(event)
         
         // AND wait for a few seconds for the events to be processed
```
Comment (edrutter-je) [inline Example/Tests/TrackerTests.swift]: No longer a few seconds.

**Review by edrutter-je — APPROVED** [review]

**Review by edrutter-je — APPROVED** [review]

**Pull request merged**